packages/fast-components-styles-msft/src/badge/index.ts

```diff
@@ -0,0 +1,108 @@
+import {
+    ComponentStyles,
+    ComponentStyleSheet,
+    CSSRules,
+} from "@microsoft/fast-jss-manager";
+import { BadgeClassNameContract } from "@microsoft/fast-components-class-name-contracts-msft";
+import {
+    applyLocalizedProperty,
+    Direction,
+    ellipsis,
+    toPx,
+} from "@microsoft/fast-jss-utilities";
+import { DesignSystem, withDesignSystemDefaults } from "../design-system/index";
+import { ensureNormalContrast } from "../utilities/colors";
+import { fontWeight } from "../utilities/fonts";
+
+function smallBadgeStyle(direction: Direction): CSSRules<DesignSystem> {
+    return {
+        padding: direction === Direction.ltr ? "0 7px 2px 0" : "0 0 2px 7px",
```

Comment:
pretty sure we have a spacing utility to do this

Reply:
Yeah, whoops. Changed.